fenix/app/src/main/res/values/preference_keys.xml

```diff
@@ -95,7 +95,7 @@
     <string name="pref_key_sign_out" translatable="false">pref_key_sign_out</string>
     <string name="pref_key_sync_sign_in" translatable="false">pref_key_sync_sign_in</string>
     <string name="pref_key_push_project_id" translatable="false">project_id</string>
-    <string name="pref_key_search_widget_installed" translatable="false">pref_key_search_widget_installed</string>
+    <string name="pref_key_search_widget_installed_2" translatable="false">pref_key_search_widget_installed_2</string>
```

Comment:
We still need to migrate the existing pref value because `setSearchWidgetInstalled` is only ever called when the widget is installed https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/main/java/org/mozilla/gecko/search/SearchWidgetProvider.kt#40

Reply:
Thank for catching it, PR updated :)